.changeset/smooth-chicken-wash.md

```diff
@@ -2,4 +2,106 @@
 'astro': minor
 ---
 
-Implements Content Layer
+Adds experimental support for the Content Layer API.
+
+The new Content Layer API builds upon content collections, taking them beyond local files in `src/content/` and allowing you to fetch content from anywhere, including remote APIs. These new collections work alongside your existing content collections, and you can migrate them to the new API at your own pace. There are significant improvements to performance with large collections of local files. 
+
+### Getting started
+
+To try out the new Content Layer API, enable it in your Astro config:
+
+```js
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+  experimental: {
+    contentLayer: true
+  }
+})
+```
+
+You can then create collections in your `src/content/config.ts` using the Content Layer API.
```

Comment:
```suggestion
You can then create collections in your `src/content/config.ts` the Content Layer API in addition to continuing to use the existing content collections API. You can configure on a per-collection basis, and your schema may contain both kinds of collections at once.
```
Because this is abridged, we don't show the "migrate to / from" nor do we mention that you can keep your existing collections if you don't want to move them all over immediately. So I think frontloading this idea makes sense here for more confident adoption!

Reply:
I've included a similar sentence in the first paragraph, so I'm not sure this is needed too.